pallets/bridges/nomad/da-bridge/src/mock.rs

```diff
@@ -1,58 +1,52 @@
-use avail_core::header::Header;
-use frame_support::{traits::GenesisBuild, weights::Weight};
+use frame_support::{parameter_types, traits::ConstU32, weights::Weight};
 use frame_system::{self as system, header_builder::da, test_utils::TestRandomness};
 use nomad_base::NomadBase;
 use sp_core::{H160, H256};
 use sp_runtime::{
-	traits::{BlakeTwo256, ConstU32, IdentityLookup},
-	AccountId32,
+	traits::{BlakeTwo256, IdentityLookup},
+	AccountId32, BuildStorage,
 };
 
 use crate::{self as da_bridge};
 
 // type TestXt = sp_runtime::testing::TestXt<Call, SignedExtra>;
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
-pub type Block = frame_system::mocking::MockBlock<Test>;
-pub type BlockNumber = u32;
+type Block = frame_system::mocking::MockDaBlock<Test>;
 
 // TODO: add proper config once frame executive mocking has been demonstrated
 // Configure a mock runtime to test the pallet.
 frame_support::construct_runtime!(
-	pub enum Test where
-		Block = Block,
-		NodeBlock = Block,
-		UncheckedExtrinsic = UncheckedExtrinsic,
+	pub struct Test
 	{
-		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		System: frame_system::{Pallet, Call, Config<T>, Storage, Event<T>},
```

Comment:
Would something like this work?
```diff
- System: frame_system::{Pallet, Call, Config<T>, Storage, Event<T>},
+ System: frame_system,
```

Reply:
Correct, this is something which we should do across the codebase to make it consistent but can be taken in separate PR specific to tests which handle the pallet defaults, executive pallet tests etc.,